### ISSUE TITLE
issue on import swifter

### ISSUE THREAD

**Issue by furbyman** [googlecolab/colabtools#358]
hello

on 
`!pip install --upgrade pandas`
`!pip install --upgrade swifter`

then 
`import pandas as pd`
`import swifter`

got error:

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-10-fa3a4d458759> in <module>()
      1 import pandas as pd
----> 2 import swifter

/usr/local/lib/python3.6/dist-packages/swifter/__init__.py in <module>()
----> 1 from .swifter import SeriesAccessor, DataFrameAccessor
      2 
      3 __all__ = ['SeriesAccessor, DataFrameAccessor']
      4 __version__ = '0.272'

/usr/local/lib/python3.6/dist-packages/swifter/swifter.py in <module>()
     12 
     13 
---> 14 @pd.api.extensions.register_series_accessor("swifter")
     15 class SeriesAccessor:
     16     def __init__(self, pandas_series, npartitions=None, dask_threshold=1, progress_bar=True):

AttributeError: module 'pandas.api' has no attribute 'extensions'

work very well on other environment (anaconda+spyter+py3...)
colabtools issue or swifter issue?

**Comment by craigcitro** [comment]
tl;dr: restart your runtime (ctrl-m . or `Runtime` -> `Restart runtime ...`).

The issue here is that pandas is loaded at colab startup, and python imports are idempotent: the `import pandas as pd` call you're making is a noop. If you restart the process, it'll pick up the newly installed version.
